apps/webmin.sh

```diff
@@ -5,6 +5,10 @@
 # shellcheck disable=2034,2059
 true
 SCRIPT_NAME="Webmin"
+SCRIPT_EXPLAINER="Webmin is a web-based interface for system administration for Unix.
+Using any modern web browser, you can setup user accounts, Apache, DNS, file sharing and much more.
+Webmin removes the need to manually edit Unix configuration files like /etc/passwd, and lets you manage a system from the console or remotely.
+See the following page with standard modules for a list of all the functions built into Webmin: https://webmin.com/standard.html"
```

Comment:
This description is from the official webmin website. Do you think that we can use that?
https://webmin.com/ (section What is Webmin?)

Reply:
LGTM